CMakeLists.txt

```diff
@@ -97,10 +118,36 @@ install(
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
 )
 
+# packaging
+
+if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+  set(CPACK_PACKAGE_CONTACT "Frank Imerson <frank.imerson@avidbots.com>")
```

Comment:
Nit picky but it's Imeson not Imerson

Reply:
Oops, that's pretty important. Just fixed it.